src/include/planner/logical_plan/scan/logical_scan_file.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "binder/copy/bound_file_scan_info.h"
+#include "planner/logical_plan/logical_operator.h"
+
+namespace kuzu {
+namespace planner {
+
+class LogicalScanFile : public LogicalOperator {
```

Comment:
I would name this as `LogicalReader` for now to be consistent with the physical operator. If you make this change, be sure to rename `boundFileScan`, `boundFileScanInfo`, and logical operator type accordingly, too.

Reply:
This PR is not the best fit for renaming (after remaining the diff will be hard to read). I'll do in a separate PR.